test/test_async.py

```diff
@@ -82,7 +83,8 @@ async def users(request):
 async def test_(request, aiohttp_client, app, case):
     request.config.HYPOTHESIS_CASES += 1
     client = await aiohttp_client(app)
-    response = await client.request(case.method, f"/v1{case.formatted_path}", headers=case.headers, json=case.body)
+    headers = case.headers or {"User-Agent": USER_AGENT}
```

Comment:
Changes in this test don't seem relevant. Please, remove them or let me know if I missed something